docs/user-guide/README.md

```diff
@@ -64,6 +64,9 @@ If you don't have much familiarity with Kubernetes, we recommend you read the fo
 [**Overview**](overview.md)
 : A brief overview of Kubernetes concepts.
 
+**Cluster**
+: A cluster is the compute resources on top of which your containers are built.
```

Comment:
Fair enough. 

I'd use a definition similar to that used by GKE: https://cloud.google.com/container-engine/docs/clusters/

A cluster is a set of physical or virtual machines and other infrastructure resources used by Kubernetes to run your applications.

I'd change the definition in the top-level README.md, also. We don't "build" containers, so that doesn't even make sense.

We should define node, as well, then.

A node is a physical or virtual machine running Kubernetes, onto which pods can be scheduled.

Both terms should link to docs. Node can link to ../admin/node.md. Cluster doesn't have a great doc, but maybe ../admin/README.md.